org.adempiere.base.process/src/org/compiere/process/RequestEMailProcessor.java

```diff
@@ -457,14 +463,26 @@ protected void createRequest(EmailContent emailContent, String trxName) throws E
 		
 		if (log.isLoggable(Level.INFO)) log.info("created request " + req.getR_Request_ID() + " from msg -> " + emailContent.subject);
 		
-		String htmlContent = emailContent.getHtmlContent(true);
-		if (htmlContent != null){
-			MAttachment attach = req.createAttachment();
-			
-			attach.addEntry(emailContent.subject + ".html", emailContent.getHtmlContent(true).getBytes(Charset.forName("UTF-8")));
-			attach.saveEx(trxName);
+		if(p_HTMLAttachmentType.contains("H")) {

```

Comment:
what's the expected value here ? if it is "H" or "I", this should be if ("H".equals(p_HTMLAttachmentType)), same goes for the comparison of "I" below (that should  be an else if instead of just if). of course, the current code is correct if your expected value here can be "HI" or "IH".

Reply:
I expected there could be variant with both values. We did not add this variant at the end. I can change it to equals.

Reply:
Changed to equals.